### ISSUE TITLE
[🚀 Feature]: selenium manager  allows  customization DRIVER_URL 

### ISSUE THREAD

**Issue by dongfangtianyu** [SeleniumHQ/selenium#12388]
### Feature and motivation

Selenium Manager is very cool and provides a unified way to obtain drivers. 
However, it currently does not allow users to obtain drivers from custom servers, which means it must work in an interconnected internet environment.

Although Selenium Manager supports proxies, the use of HTTPS protocol in the URL makes the mirror server unable to function.

### Usage example

If DRIVER_URL allows customization through `command line parameters` or `environment variables`, then I can start an HTTP server and search for the corresponding driver in the mirror based on the URL.

The working method of Selenium Manager will not change, but it provides great help for users in isolated networks or networks that cannot access Google and GitHub

**Comment by MILADZASH** [comment]
hi. from this code:
# Import necessary modules
from selenium import webdriver
from selenium.webdriver.firefox.options import Options
from bs4 import BeautifulSoup
from datetime import datetime, timedelta
from decimal import Decimal
import csv
import sys
import time
import requests
import pandas as pd
import pytz
import os
########################################################################################
# Set up the driver
options = webdriver.FirefoxOptions()
options.add_argument("--no-sandbox")
options.add_argument("--disable-dev-shm-usage")
options.add_argument("--disable-gpu")
options.add_argument("--disable-extensions")
options.add_argument('-headless')  # run Firefox in headless mode
driver = webdriver.Firefox(options=options)

i have this error:
WARNING:selenium.webdriver.common.selenium_manager:The version of firefox cannot be detected. Trying with latest driver version

---------------------------------------------------------------------------

WebDriverException                        Traceback (most recent call last)

[<ipython-input-40-04de7e1eabe5>](https://localhost:8080/#) in <cell line: 23>()
     21 options.add_argument("--disable-extensions")
     22 options.add_argument('-headless')  # run Firefox in headless mode
---> 23 driver = webdriver.Firefox(options=options)
     24 #########################################################################################
     25 #                                   URL

4 frames

[/usr/local/lib/python3.10/dist-packages/selenium/webdriver/remote/errorhandler.py](https://localhost:8080/#) in check_response(self, response)
    243                 alert_text = value["alert"].get("text")
    244             raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here
--> 245         raise exception_class(message, screen, stacktrace)

WebDriverException: Message: Process unexpectedly closed with status 1

whats problem Bro ?

**Comment by titusfortner** [comment]
@dongfangtianyu can you give logs of what you are doing and what isn't working? https://www.selenium.dev/documentation/webdriver/troubleshooting/logging/

**Comment by titusfortner** [comment]
Or grab the current version from here https://github.com/SeleniumHQ/selenium/tree/trunk/common/manager/
And execute what you want to work that doesn't with `--debug`?

**Comment by titusfortner** [comment]
@MILADZASH that looks like a completely separate issue, please make sure you have Firefox downloaded on your system and create a new issue rather than continuing here.

**Comment by MILADZASH** [comment]
@titusfortner im very confuse and i dont know where i must say my problem bro

**Comment by diemol** [comment]
This is already possible https://www.selenium.dev/documentation/selenium_manager/ `SE_DRIVER_MIRROR`

**Comment by dongfangtianyu** [comment]
> This is already possible https://www.selenium.dev/documentation/selenium_manager/ `SE_DRIVER_MIRROR`

@diemol   thank you, this looks exactly what I'm looking for.

But it is currently not working properly, still downloading from the original HTTPS connection.

**Debug info**
```
C:\Users\tianyu\Desktop>set | findstr SE_DRIVER_MIRROR                                                            
SE_DRIVER_MIRROR=http://127.0.0.1:8888

C:\Users\tianyu\Desktop>selenium-manager.exe --browser firefox --debug
DEBUG   Using shell command to find out firefox version
DEBUG   Running command: "set PFILES=%PROGRAMFILES: (x86)=%&& wmic datafile where name='!PFILES:\\=\\\\!\\\\Mozilla Firefox\\\\firefox.exe' get Version /value"
DEBUG   Output: "\r\r\n\r\r\nVersion=95.0.2.8022\r\r\n\r\r\n\r\r\n\r"
DEBUG   The version of firefox is 95.0.2.8022
DEBUG   Detected browser: firefox 95
ERROR   error sending request for url (https://github.com/mozilla/geckodriver/releases/latest): error trying to connect: tcp connect error:  (os error 10060)
```


**Version info**
```
C:\Users\Tianyu\Desktop>pip show selenium
Name: selenium
Version: 4.10.0
Summary:
Home-page: https://www.selenium.dev
Author:
Author-email:
License: Apache 2.0
Requires: certifi, trio, trio-websocket, urllib3

C:\Users\Tianyu\Desktop>selenium-manager.exe -V
selenium-manager 1.0.0-M3
```

Is my usage incorrect?
Kindly request experienced individuals to share more information. Thank you.

**Comment by titusfortner** [comment]
@bonigarcia can you check on this?

**Comment by bonigarcia** [comment]
That feature is planned, but it is not yet available on the Rust side:

https://github.com/SeleniumHQ/selenium/issues/11687

`SE_DRIVER_MIRROR` should not be in the documentation yet. I want to enhance the documentation page of Selenium Manager by next month.

**Comment by titusfortner** [comment]
Thanks. I've removed it from the docs, and we'll track this issue there.
